ee/ucp/admin/install/system-requirements.md

```diff
@@ -59,6 +59,15 @@ indicated as the "Scope" of that port. The three scopes are:
 - Internal: Traffic arrives from other hosts in the same cluster.
 - Self: Traffic arrives to that port only from processes on the same host.
 
+> Note
+>
+> When installing UCP on to Microsoft Azure, an overlay network is not used for
+> Kubernetes, therefore any containerized service deployed onto Kubernetes and
+> exposed as a Kubernetes Service may need its corresponding port to be opened
+> on the underlying Azure Network Security Group. For more information see
+> [Installing on
+> Azure](/ee/ucp/admin/install/cloudproviders/install-on-azure/#azure-prerequisites).
+
```

Comment:
Suggest: "When installing UCP on Microsoft Azure, an overlay network is not used for Kubernetes; therefore, any containerized service deployed onto Kubernetes and exposed as a Kubernetes Service may need its corresponding port opened on the underlying Azure Network Security Group. For more information, see Installing on Azure."